smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/AuthSchemeMiddleware.kt

```diff
@@ -0,0 +1,37 @@
+package software.amazon.smithy.swift.codegen.integration.middlewares
```

Comment:
The code generator that constructs & inserts `AuthSchemeMiddleware` to operation stack in generated client code.